Dockerfile

```diff
@@ -1,49 +1,42 @@
-# To run: docker run -v /path/to/wsgi.py:/var/www/indexd/wsgi.py --name=indexd -p 81:80 indexd
-# To check running container: docker exec -it indexd /bin/bash
+ARG AZLINUX_BASE_VERSION=master
 
-FROM quay.io/cdis/python:python3.9-buster-2.0.0
+## For local development
+# FROM quay.io/cdis/python-nginx-al2:feat_python-nginx AS base
+
+FROM 707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/python-nginx-al2:feat_python-nginx as base
```

Comment:
let's follow the warning and fix `The 'as' keyword should match the case of the 'from' keyword`